cmd/flags.go

```diff
@@ -39,6 +47,10 @@ const (
 	defaultDebugAddr = "localhost:7597"
 )
 
+// lineBreak can be included in a command list to provide a blank line
+// to help with readability
+var lineBreak = &cobra.Command{Run: func(*cobra.Command, []string) {}}
```

Comment:
This should be inlined into the `lineBreakCommand` function, as that should be the only reference to the command; and then we won't have to dereference a pointer to get an instance of the command.